ui/src/app/app-routing.module.ts

```diff
@@ -322,7 +320,11 @@ const routes: Routes = [
       {
         path: 'admin',
         children: [
-          // legacy / duplicated routes go HERE
+          {
+            path: 'banner',
+            component: AppRouting,
+            data: {title: 'Create Banner'}
```

Comment:
Does `title: 'Create Banner'` need to be set here?

Reply:
Oh, dang, actually that needs to go away. Thanks!